src/chrome/content/rules/T-Mobile.xml

```diff
@@ -97,70 +94,36 @@ Fetch error: http://find.my.t-mobile.com/ => https://find.my.t-mobile.com/: (6,
 	* Secured by us
 
 -->
-<ruleset name="T-Mobile.com (partial)" default_off='failed ruleset test'>
-
-	<!--	Direct rewrites:
-				-->
+<ruleset name="T-Mobile.com (partial)">
 	<target host="t-mobile.com" />
+	<target host="www.t-mobile.com" />
 	<target host="account.t-mobile.com" />
 	<target host="es.account.t-mobile.com" />
-	<target host="b2b.t-mobile.com" />
 	<target host="business.t-mobile.com" />
-
-	<!--target host="cache.business.t-mobile.com" /-->
 	<target host="es.business.t-mobile.com" />
 	<target host="find.business.t-mobile.com" />
-
 	<target host="ebill.t-mobile.com" />
 	<target host="es.t-mobile.com" />
 	<target host="find.es.t-mobile.com" />
 	<target host="explore.t-mobile.com" />
-	<!--target host="cache.explore.t-mobile.com" /-->
 	<target host="es.explore.t-mobile.com" />
-	<target host="find.t-mobile.com" />
 	<target host="my.t-mobile.com" />
-
 	<target host="account.my.t-mobile.com" />
 	<target host="es.account.my.t-mobile.com" />
 	<target host="es.my.t-mobile.com" />
-	<target host="find.my.t-mobile.com" />
 	<target host="m.my.t-mobile.com" />
 	<target host="manage.my.t-mobile.com" />
 	<target host="es.manage.my.t-mobile.com" />
-
 	<target host="newsroom.t-mobile.com" />
 	<target host="prepaid-phones.t-mobile.com" />
-	<!--target host="cache.prepaid-phones.t-mobile.com" /-->
 	<target host="es.prepaid-phones.t-mobile.com" />
 	<target host="secure-checkout.t-mobile.com" />
 	<target host="support.t-mobile.com" />
 	<target host="apps.support.t-mobile.com" />
 	<target host="es.support.t-mobile.com" />
-	<!--target host="videos.t-mobile.com" /-->
-	<!--target host="es.videos.t-mobile.com" /-->
-	<!--target host="cache.videos.t-mobile.com" /-->
```

Comment:
Can you please keep the comments or move it on top (as long as the hosts exist)?